example_relay.conf

```diff
@@ -19,7 +19,19 @@ max_concurrent: 8
 # Environment variable: $RELAY_DYNAMIC_CONFIG_ROOT
 # Default: none
 # Required: No
-# dynamic_config_root: /tmp/bundle_configs
+# dynamic_config_root: /tmp/dynamic_configs
+
+# Use managed dynamic configuration
+# Requires dynamic_config_root
+# Environment variable: $RELAY_MANAGED_DYNAMIC_CONFIG
+# Default: false
+# managed_dynamic_config: false
+
+# Refresh interval for managed dynamic configuration files.
+# Valid time units are s (seconds), m (minutes), and h (hours).
+# Requires managed_dynamic_config
+# Environment variable: $RELAY_MANAGED_DYNAMIC_CONFIG_INTERVAL
+# Default: 5s
```

Comment:
We'll need to add these to the doc site: http://docs.operable.io/docs/relay-environment-variables